mlflow/metrics/genai/model_utils.py

```diff
@@ -146,13 +154,166 @@ def _call_openai_api(openai_uri, payload, eval_parameters, extra_headers, proxy_
 
 
 _PREDICT_ERROR_MSG = """\
-Failed to call the deployment endpoint. Please check the deployment URL\
+Failed to call the deployment endpoint. Please check the deployment URL \
 is set correctly and the input payload is valid.\n
 - Error: {e}\n
 - Deployment URI: {uri}\n
 - Input payload: {payload}"""
 
 
+def _is_supported_llm_provider(schema: str) -> bool:
```

Comment:
Seems this function can be reused, could we put it inside provider_registry directly?

Reply:
Sounds good:)